lab_1/main.py

```diff
@@ -25,9 +35,14 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
-    pass
+    for word in tokens:
+        if word in stop_words:
+            tokens.remove(word)
```

Comment:
try this:
stop_words = ['the', 'is']
tokens = ['the', 'is', 'a', 'the']

Does it work correctly?